feature-governance-impl/src/main/java/io/novafoundation/nova/feature_governance_impl/presentation/tindergov/cards/TinderGovCardsViewModel.kt

```diff
@@ -203,10 +190,23 @@ class TinderGovCardsViewModel(
             }
 
             interactor.addItemToBasket(referendum.id, voteType)
+            checkAllReferendaWasVotedAndOpenBasket()
+
             isVotingInProgress.value = false
         }
     }
 
+    private suspend fun checkAllReferendaWasVotedAndOpenBasket() {
+        val basket = interactor.getTinderGovBasket()
+            .associateBy { it.referendumId }
```

Comment:
JFYI if we only need to check for inclusion we can convert collection to a set (`mapToSet { it.referendumId }` in this case) instead of converting collection to a map
This is minor though